packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -253,21 +254,31 @@ def _matplotlib_color_to_CSS(self, color, alpha=None, is_RGB=True):
             R = int(color[0] * 255)
             G = int(color[1] * 255)
             B = int(color[2] * 255)
-            CSS_color = """rgba({0:d}, {1:d},
-                                {2:d}, {3:.3g})""".format(R, G, B, alpha)
+            if len(color) == 3 or alpha_overrides:
+                CSS_color = """rgba({0:d}, {1:d},
```

Comment:
Let's put the string on one line -- this will actually introduce a bunch of whitespace into the CSS string that the browser then has to parse out.

Reply:
This is primarily done to adhere with `flake8`, what else can be done?